packages/core/src/components/AdaptiveGrid.story.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import AdaptiveGrid from './AdaptiveGrid';
+
+const gridItem = (
+  <div style={{ height: '100%', border: '1px solid black', padding: '16px' }}>Grid Item</div>
+);
+const bigGridItem = (
+  <div style={{ height: '100%', border: '1px solid black', padding: '16px' }}>
+    Very Big Grid Item Very Big Grid Item Very Big Grid Item Very Big Grid Item Very Big Grid Item
```

Comment:
Use `<LoremIpsum />` component here?

Reply:
it has a `short` prop too for smaller ipsum